Avatar/Avatar.css

```diff
@@ -0,0 +1,56 @@
+body {
+    font-family: sans-serif;
+  }
+  .wrapper
+  {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;  
+    gap: 2rem;
+  }
+  
+  .avatar-container {
+    align-items: center;
+    background-color: #f0f0f0;
+    border-radius: 0.3rem;
+    display: flex;
```

Comment:
Add display flex above other flex properties. Makes the code easier to read.

Reply:
I have added them in alphabetical order. Hoping its not an issue. Please approve.